libnavigation-util/src/main/java/com/mapbox/navigation/utils/network/NetworkStatusService.kt

```diff
@@ -26,40 +25,36 @@ class NetworkStatusService(private val applicationContext: Context) {
     fun getNetworkStatusChannel(): ReceiveChannel<NetworkStatus> = networkStatusChannel
 
     init {
-        @TargetApi(11)
-        when (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            true -> {
-                val builder = NetworkRequest.Builder()
-                    .addCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET)
-                    .addTransportType(NetworkCapabilities.TRANSPORT_CELLULAR)
-                    .addTransportType(NetworkCapabilities.TRANSPORT_WIFI)
-                    .addTransportType(NetworkCapabilities.TRANSPORT_VPN)
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
We should revert these changes, they're not affected by bumping the `minSdkVersion` to `19`.

Reply:
made it to restrict `@Target` annotation

Reply:
What's the problem with `@Target` annotation?

Reply:
it's just not necessary here, seems clean without this annotation